catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -106,7 +106,7 @@ public boolean touchDown(InputEvent event, float x, float y, int pointer, int bu
 		this.addListener(new BroadcastListener() {
 			@Override
 			public void handleBroadcastEvent(BroadcastEvent event, String broadcastMessage) {
-				doHandleBroadcastEvent(broadcastMessage);
+				doHandleBroadcastEvent(event, broadcastMessage);
```

Comment:
event is handed through a few functions just to be used for getting the map (event.getSenderSprite().getBroadcastWaitSequenceMap())
maybe pass the map through?